src/main/java/edu/cornell/kfs/sys/batch/service/impl/TablesPurgeServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package edu.cornell.kfs.sys.batch.service.impl;
+
+import java.util.Date;
+
+import org.kuali.kfs.coreservice.framework.parameter.ParameterService;
+
+import edu.cornell.kfs.sys.batch.service.TablesPurgeService;
+
+public abstract class TablesPurgeServiceImpl implements TablesPurgeService {
+    
+    protected static final String DEFAULT_NAME_SPACE_CODE = "KFS-SYS";
+    protected static final String DEFAULT_COMPONENT = "PurgeTablesStep";
+    protected static final String DEFAULT_PARAMETER_NAME = "DEFAULT_NUMBER_OF_DAYS_OLD";
+    
+    protected ParameterService parameterService;
+    
+    public abstract void purgeRecords(Date jobRunDate); 
+    
+    protected int retrieveDefaultDaysBeforePurgeParameterValue() {
```

Comment:
Is there a way to override the default? Is there going to be a specific param for some of the jobs?